commodore/dependency_mgmt.py

```diff
@@ -96,21 +96,13 @@ def _read_component_urls(cfg: Config, component_names) -> Dict[str, str]:
             f"Could not read Commodore configuration: {e}"
         ) from e
 
-    for component_name in component_names:
-        component_urls[
-            component_name
-        ] = f"{cfg.default_component_base}/{component_name}.git"
-
-    if commodore_config is None:
-        click.secho("Empty Commodore config file", fg="yellow")
-    else:
-        for component_override in commodore_config.get("components", []):
-            if cfg.debug:
-                click.echo(
-                    f"   > Found override for component {component_override['name']}:"
-                )
-                click.echo(f"     Using URL {component_override['url']}")
-            component_urls[component_override["name"]] = component_override["url"]
+    for component in commodore_config.get("components", []):
```

Comment:
I would switch the loop to iterate through `component_names` and lookup the URL in `commodore_config`. But it doesn't really matter I guess, it's just my preference...

Reply:
Actually, it is done this way. But`commodore_config` provides an array. By doing as you suggest, I would need to iterate it for each component in `component_names`. To mitigate this, the input from `commodore_config` is converted to a dict within this loop.